### DISCUSSION TITLE
Export more than 24 hour log

### DISCUSSION THREAD

**Discussion by albgen** [datalust/seq-tickets#2319] [Help] [answered]
Hello,

need to export all the logs from SEQ. Can i do this?
Currently i see only export for 24 hours which is not enough.

thank you

**Comment by nblumhardt** [comment, marked as answer]
Hi Evald; the best way to export large numbers of events from Seq is using `seqcli search --json` and piping the results to a file. You'll still need to specify a count, and I'd suggest doing it one day at a time if there's a lot of data to get through.

```
seqcli search --json -c 1000000000 --start="2024-01-01T00:00:00Z" --end="2024-01-02T00:00:00Z" > logs-2024-01-01.ndjson
```

Hope this helps,
Nick

**Comment by albgen** [comment]
hi, 

thanks  that worked.
